cockatrice/src/dlg_connect.cpp

```diff
@@ -92,17 +112,16 @@ DlgConnect::DlgConnect(QWidget *parent)
     QGroupBox *loginGroupBox = new QGroupBox(tr("Login"));
     loginGroupBox->setLayout(loginLayout);
 
+	QGroupBox *btnGroupBox = new QGroupBox(tr(""));
```

Comment:
Ill try it.  I dont recall if it threw errors or not when you left out the group box title name.